Hanlding Exceptions

try-except equals to java try-catch
```python
>>> while True:
...   try:
...     x = int(input("Input a number:"))
...     break;
...   except ValueError:
...     print("Invalid number")
...
Input a number:kk
Invalid number
Input a number:2a
Invalid number
Input a number:12
```
